prow/cmd/jenkins-operator/BUILD

```diff
@@ -2,6 +2,18 @@ package(default_visibility = ["//visibility:public"])
 
 licenses(["notice"])
 
+load("@io_bazel_rules_docker//go:image.bzl", "go_image")
+
+go_image(
+    name = "image",
+    base = "@alpine-base//image",
+    embed = [":go_default_library"],
+    importpath = "k8s.io/test-infra/prow/cmd/jenkins-operator",
+    pure = "on",
+    race = "off",
+    tags = ["automanaged"],
```

Comment:
oh, I guess this was missed in a cleanup. we shouldn't have `automanaged` tags on any go rules anymore...

Reply:
Removed.